pollux/lib/anoncreds/src/main/scala/io/iohk/atala/pollux/anoncreds/AnoncredLib.scala

```diff
@@ -159,8 +163,17 @@ object AnoncredLib {
       )
   }
 
+  // TODO FIX
```

Comment:
@CryptoKnightIOG, can you create a test for this issue and a ticket in the jira or add any other documentation that describes this issue?

Reply:
This was added by Fabio. Will follow up with him.